NRAVPlayerTracker/NRAVPlayerTracker/Tracker/NRTrackerAVPlayer.m

```diff
@@ -215,6 +215,11 @@ - (void)observeValueForKeyPath:(NSString *)keyPath
         if (!self.state.isBuffering && self.state.isPaused && self.playerInstance.rate == 0.0) {
             [self sendSeekStart];
         }
+        // The state isSeekingDuringPlayback can be set manually using the setIsSeekingDuringPlayback function
+        // Otherwise, KVO observers are not being notified that a seek happened during playback (rate == 1.0)
+        else if (!self.state.isBuffering && self.state.isSeekingDuringPlayback && !self.state.isPaused && self.playerInstance.rate == 1.0) {            
```

Comment:
Is the value `rate` equivalent to the playback speed? If so, should we simply do a > 0 ?

Reply:
Indeed, the `AVPlayer.rate` can be _almost_ anything between `0.5` and `2.0`.

Reply:
Je pensais qu'on aurait pas besoin de cette condition là et qu'on se serait simplement basé sur la modification de `isSeekingDuringPlayback` pour lancer un `[self sendSeekStart];`. En rendant `isSeekingDuringPlayback` compatible KVO est-ce que ça fonctionnerait?